avr/cores/megacommand/MCL/GridPage.cpp

```diff
@@ -622,17 +622,18 @@ void GridPage::apply_slot_changes() {
 
   MDTrack md_track;
   MDSeqTrack md_seq_track;
-  for (uint8_t track = 0; track < count && track + getCol() < 20; track++) {
+  for (uint8_t track = 0; track < count && track + getCol() < NUM_TRACKS; track++) {
     if (slot_clear == 1) {
       grid.clear_slot(track + getCol(), getRow());
       reload_slot_models = true;
     } else if (slot_copy == 1) {
       SET_BIT32(slot_buffer_mask, track + getCol());
+    } else {
+      slot.active = row_headers[cur_row].track_type[track + getCol()];
+      //  if (slot.active != EMPTY_TRACK_TYPE) {
+      slot.store_track_in_grid(track + getCol(), getRow());
+      proj.file.sync();
```

Comment:
proj.file.sync() should be outside bracket. or also included in if (slot_clear == 1)

Reply:
done